### ISSUE TITLE
Custom build

### ISSUE THREAD

**Issue by svipas** [be5invis/Iosevka#1158]
Hi @be5invis, I'm not very familiar with font building from source and I would like to ask for your help. How I can build Iosevka with custom options, what I really want is this:

- Get all weights for Iosevka & Iosevka Term
- Use `ss04` (Menlo Style) for both Iosevka & Iosevka Term
- By default enable `dlig` for both Iosevka & Iosevka Term
- Use 450 weight for `Regular` & `Oblique` & `Italic` for both Iosevka & Iosevka Term

Basically what I want is to simply have Iosevka SS04 & Iosevka Term SS04 with enabled `dlig` by default and 450 weight for `Regular` & `Oblique` & `Italic` (because it looks too thin for me).

I hope you can help me. Thanks a lot!

**Comment by svipas** [comment]
I guess I managed to make a custom config and I think it will work, this website helped me: https://typeof.net/Iosevka/customizer

Sorry for disturbing you, I should do a little more research before creating ticket...

But what I don't understand is what is the difference between `Normal` and `FontConfig Mono`? Which one I should use? Which one is used by Iosevka SS04 when I download it from "Releases" page? I'm using your font in IDE.

**Comment by svipas** [comment]
Okay I guess I found everything I needed and I think I see it simply uses `Normal` spacing. Now it's time to try and build the custom font.
